app/controllers/analysis_controller.rb

```diff
@@ -0,0 +1,420 @@
+# Analysis controller
+require 'json'
+
+class AnalysisController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    verify_is_admin
+  end
+
+=begin Go through all of the death records in Nightingale searching for
```

Comment:
This is one way of doing multi-line comments, and isn't wrong, but the style we typically have been using throughout the project is more like:

```
# This is a multi-line comment.
# This is the second line of the comment.
# This is the third line of the comment.
```

The number of columns looks good though, definitely want that at least under 100.